airflow/gcp/hooks/cloud_storage_transfer_service.py

```diff
@@ -418,7 +421,7 @@ def _inject_project_id(self, body: Dict, param_name: str, target_key: str) -> Di
     @staticmethod
     def operations_contain_expected_statuses(
         operations: List[Dict],
-        expected_statuses: Union[Tuple[str], str]
+        expected_statuses: Union[Set[str], str]
```

Comment:
Probably it's a good idea to create issue to unify expected statuses in all GCP operators to be optional an in case of `None` assign a default set. 

FYI I talked with @TobKed about that and I think it's better to use set than tuple, but set could not be default value as it's a mutable structure.

Reply:
@nuclearpinguin is it not the only place, except `GCPTransferServiceWaitForJobStatusSensor` where it is necessary? If yes it can be done in next PR for sensors which will be dependent on this one. WDYF?

Reply:
Ok, I have a feeling that this was more popular.